### ISSUE TITLE
Mega Drive 3 button controller and CRT Royale compatibility.

### ISSUE THREAD

**Issue by andybrito** [OpenEmu/OpenEmu#2095]
I bought a Retrode 2 hoping to use the SNES and MD controllers with OpenEmu, only after receiving it i've discovered that it wasn't compatible for MD controllers, only with 1 SNES controller.

I would like to know if at least one of the MD ports on Retrode 2 will work on the new version of OpenEmu and if not, which MD adapter is known to work with OpenEmu, maybe this one ? 
http://www.ebay.com/itm/May-Flash-Mega-Drive-MD-Genesis-USB-Controller-Adapter-for-PC-Windows-Mac-/251832222990?hash=item3aa25ec50e

Also, i really loved the CRT Royale shader running on Retroarch, really great shader, but after many tries, it looks like there's something that doesn't let this filter (and a bunch of others cgp filters) run properly. Will those complex cgp filters supported in the new version ? 

I can't wait to try the new version, i am always checking the new comitts waiting the newest update, any estimate of when you guys will release it ?

**Comment by clobber** [comment]
Retrode: you probably need to locally remove the entry from the database https://github.com/OpenEmu/OpenEmu/blob/master/OpenEmu/Controller-Database.plist#L1604

Shader: #1824 You don't need to open another ticket for this. It is likely not compatible and probably won't be. The Cg shader spec will be removed in a future.

**Comment by andybrito** [comment]
I though the issue with crt lottes was purely a scaling thing since the filter works perfectly with PSP games, CRT Royale never worked under any circumstance, i'm no expert in anything but i know previous versions of Retroarch were not compatible with what crt royale seems to require to work properly, but nevermind i have 4 cg filters that i love working perfectly... to be honest i am concerned with your last phrase "The Cg shader spec will be removed in a future", this means you will remove support for .cg filters in the future ?

**Comment by andybrito** [comment]
Holy c***, the MD controller is now working... thanks a lot clobber!

**Comment by clobber** [comment]
What I meant was, in the future, we'll switch back to GLSL since Nvidia no longer supports Cg. Our current set of shaders bundled with the app will just have to be rewritten but will still be available.

**Comment by andybrito** [comment]
Retroarch is not very intuitive, openemu has the right vibe for retrogaming and these complex filters/shaders is the only thing i would like to see being supported eventually, anyway, thanks a lot for your help with Retrode.
